recipes/gcc/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "10.1.0":
```

Comment:
It has been a while since you wrote this (thanks!), we might want to update this to `10.2.0` before merging.

Reply:
yeah, let's try to add new release and update URL to an official mirror